contrib/babelfishpg_tsql/sql/sys_functions.sql

```diff
@@ -1673,11 +1673,11 @@ value	sys.sql_variant
 as $$
 begin
 -- currently only support COLUMN property
-IF (((SELECT coalesce(property_name, '')) = '') or
-    ((SELECT coalesce(property_name, '')) = 'COLUMN')) THEN
-	IF (((SELECT coalesce(level0_object_type, '')) = 'schema') and
-	    ((SELECT coalesce(level1_object_type, '')) = 'table') and
-	    ((SELECT coalesce(level2_object_type, '')) = 'column')) THEN
+IF (((SELECT coalesce(property_name COLLATE "C", '')) = '') or
```

Comment:
None of these SELECTs are needed; execution is implied by the IF.

Reply:
Removed